assets/wire/components/ItemDetails.jsx

```diff
@@ -6,10 +6,13 @@ import PreviewTags from './PreviewTags';
 import ListItemPreviousVersions from './ListItemPreviousVersions';
 import ListItemNextVersion from './ListItemNextVersion';
 import { gettext, fullDate, formatHTML } from 'utils';
-import { getPicture, getDetailRendition, showItemVersions, getCaption, isKilled, DISPLAY_ABSTRACT } from 'wire/utils';
+import { getPicture, getDetailRendition, showItemVersions, getCaption, isKilled, DISPLAY_ABSTRACT, isPreformatted } from 'wire/utils';
 
 function ItemDetails({item, user, actions, onClose}) {
     const picture = getPicture(item);
+    const itemType = isPreformatted(item)?'preformatted':'text';
```

Comment:
imo there should be spaces around `?` and `:` not sure why eslint doesn't complain